From cc1a772e51dee3562503067d631fdb0aad6b9590 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Tue, 1 Nov 2016 17:59:27 +0200
Subject: [PATCH 1065/2241] dts: cp110: add comphy nodes

This patch adds new comphy nodes with the required properties
to cp110 device tree.

Binding for this patch is availalbe in the following patch:
"dt-bindings: phy: add Marvell Comphy binding".

Change-Id: I879ee5e4ec13642d2e8f5b5cf5324a16de98f91a
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33528
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi | 8 ++++++++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   | 8 ++++++++
 2 files changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index bf506c9..68f208b 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -320,3 +320,11 @@ eip197@800000 {
 	clocks = <&cps_syscon0 1 26>;
 	status = "disabled";
 };
+
+comphy1: comphy {
+	compatible = "marvell,cp110-comphy";
+	reg = <0x441000 0x100>, <0x120000 0x6000>;
+	reg-names = "comphy", "serdes";
+	#phy-cells = <2>;
+	status = "disabled";
+};
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index f54330c..aab16b7 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -423,3 +423,11 @@ eip197: eip197@800000 {
 	clocks = <&cpm_syscon0 1 26>;
 	status = "disabled";
 };
+
+comphy0: comphy {
+	compatible = "marvell,cp110-comphy";
+	reg = <0x441000 0x100>, <0x120000 0x6000>;
+	reg-names = "comphy", "serdes";
+	#phy-cells = <2>;
+	status = "disabled";
+};
-- 
2.7.4

